javaparser-symbol-solver-testing/src/test/test_sourcecode/javasymbolsolver_0_6_0/src/java-symbol-solver-model/com/github/javaparser/symbolsolver/model/typesystem/Wildcard.java

```diff
@@ -100,7 +100,7 @@ public String describe() {
         } else if (type == BoundType.EXTENDS) {
             return "? extends " + boundedType.describe();
         } else {
-            throw new UnsupportedOperationException();
+            throw new UnsupportedOperationException(String.format("%s is not supported", type));
```

Comment:
This is an improvement - thanks! 

Could we rephrase slightly to not use the phrase "is not supported" -- instead stating _WHAT_ isn't valid please? 

e.g. maybe `String.format("Unsupported BoundType provided: %s" type)` or something like that.

Reply:
Ideally error messages should be "actionable" -- i.e. describing how it could be fixed, not just what went wrong.

- "An exception was thrown - here's the stack trace"
- "something went wrong" 
- "the bound type of XXX is invalid" 
- "the provided bound type of XXX is invalid, only BoundType.SUPER and BoundType.EXTENDS is permitted"